### PR TITLE
Feature/sidebar

### PR DESCRIPTION
**Pull request by jphorner**
# Thank you for pushing your code!
## What (if any) features are you implementing?
+ Created basic sidebar with title, buttons, and rudimentary CSS

## What (if anything) did you refactor?
+ Changed 'section' to 'aside'

## Were there any issues that arose?
+ None yet

## Is there anything that you need from your teammate?
+ Just more of their amazing ideas!

## Any other comments, questions, or concerns?
+ May need refactor CSS to make it adaptive; need to experiment with flexbox to get the alignment correct

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<head>
+  <title>Idea Box</title>
+  <meta charset="UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://fonts.googleapis.com/css?family=EB+Garamond&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <main>
+    <aside>
```
Comment (MTCawthray) [inline index.html:12]: nice use of the aside element!

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+h1 {
+  color: #ffffff;
+  padding-left: 15px;
+}
+
+aside {
+  /* padding-left: 15px; */
+  height: 60em;
+  width: 17em;
+  margin-right: 10px;
```
Comment (MTCawthray) [inline styles.css:22]: You're right, we may want to revisit using relative sizing rather than absolute

**Review by Raquelhill — COMMENTED** [review]
We have made great progress.  We need more time experimenting with the CSS and our page will look great!

**Review by MTCawthray — APPROVED** [review]
This all looks great!

**Pull request merged**